features/step_definitions/group_selection_steps.rb

```diff
@@ -419,6 +436,31 @@
   end
 end
 
+And(/.* should see eligibility failed error on dependent with resident role/) do
+  expect(find('.ineligible_ivl_row .ivl_errors')).to have_content("eligibility failed on active_individual_role")
```

Comment:
Same feedback as the other comments-- can we avoid hardcoding the English string here by targeting some other attribute or by relying on an existing translation?

Reply:
This originates from legacy code and currently does not utilize any translations.
`
./models/insured_eligible_for_benefit_rule.rb:57:       @errors << ["eligibility failed on market kind"]
./models/insured_eligible_for_benefit_rule.rb:61:       @errors << ["eligibility failed on #{element}"]
./models/insured_eligible_for_benefit_rule.rb:87:      "eligibility failed on #{element}"
`

Reply:
Understood.  We'll have to refactor the original code in the future.  Thank you for updating the other places where possible!